cmd/root.go

```diff
@@ -0,0 +1,23 @@
+package cmd
+
+import "github.com/spf13/cobra"
+
+var rootCmd = &cobra.Command{
+	Use:   "hermione",
+	Short: "Generic Linux task running service and client",
+}
+
+func init() {
+	rootCmd.AddCommand(
+		serverCmd,
+		startCmd,
+		stopCmd,
+		statusCmd,
+		logsCmd,
```

Comment:
seems to be missing `serverRunCmd`?

```suggestion
		logsCmd,
		serverRunCmd,
```

Reply:
No, serverCmd has further sub-commands.

See [cmd/server.go:11](https://github.com/zostay/hermione/blob/02560e7a904717f3b482c1c9a95dda2edce8a9f4/cmd/server.go#L11)

Reply:
Ah, sorry, pardon my (lack of) reading comprehension!